common/OpTestUtil.py

```diff
@@ -1246,36 +1246,36 @@ def setup_term(self, system, pty, ssh_obj=None, block=0):
         if system_obj.state == 3: # OpSystemState.PETITBOOT
             return
 
-        rc = pty.expect(['login: $', ".*#$", ".*# $", ".*\$", "~>", 'Petitboot', pexpect.TIMEOUT, pexpect.EOF], timeout=10)
+        rc = pty.expect(['login: $', ".*#$", ".*# $", ".*\$", "~>", "~.*#", 'Petitboot', pexpect.TIMEOUT, pexpect.EOF], timeout=10)
```

Comment:
Why isn't ~ captured by . in the second regex ".*#$" ?

Reply:
Right.I figured out that ssh here has some special characters like `^[[1m^[[31mlinux-he6x:~ #^[(B^[[m` which does not match with the `$` end part in second and third regex. 

Can a change like  `".*#$"` -> `".*#.*$"` help here, it would also obsolete the third regex?

Reply:
please let know if anything is required here.